### PR TITLE
Add faster Laplace

### PR DESCRIPTION
**Pull request by rohanbabbar04**
## Description
- Add class `Laplace` with all methods.
- Update test in `test_scipy.py`.

## Checklist

- [x] Code style is correct (follows pylint and black guidelines)
- [x] Includes new or updated tests to cover the new feature
- [x] New features are properly documented (with an example if required)

### CODE REVIEW

File: preliz/distributions/laplace.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        Compute the percent point function (PPF) at a given probability q.
+        """
+        q = np.asarray(q)
+        return nb_ppf(q, self.mu, self.b)
+
+    def logpdf(self, x):
+        """
+        Compute the log probability density function (log PDF) at a given point x.
+        """
+        return np.log(nb_pdf(x, self.mu, self.b))
```
Comment (aloctavodia) [inline preliz/distributions/laplace.py:92]: This should be the other way around, write an explicit expression for logpdf, and exponentiate that to compute the pdf.

File: preliz/distributions/laplace.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    if q > 0.5:
+        q = -np.log(2 * (1 - q))
+    else:
+        q = np.log(2 * q)
+    return q * b + mu
+
+
+@nb.njit
+def nb_neg_logpdf(x, mu, b):
+    return (-np.log(nb_pdf(x, mu, b))).sum()
```
Comment (aloctavodia) [inline preliz/distributions/laplace.py:159]: define nb_logpdf and use it

**Review by aloctavodia — CHANGES_REQUESTED** [review]

File: preliz/distributions/laplace.py (hunk 1 above)
Comment (rohanbabbar04) [reply preliz/distributions/laplace.py]: Noted, will keep that in mind....

File: preliz/distributions/laplace.py (hunk 2 above)
Comment (rohanbabbar04) [reply preliz/distributions/laplace.py]: Done

**Review by aloctavodia — APPROVED** [review]

**Pull request merged**